src/OpenTelemetry.Exporter.OneCollector/OpenTelemetry.Exporter.OneCollector.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>An OpenTelemetry .NET exporter that sends telemetry to Microsoft OneCollector</Description>
-    <Authors>OpenTelemetry Authors</Authors>
```

Comment:
On a related topic, do we allow component owners to put different authors info here or we enforce that all projects in this repo should use "OpenTelemetry Authors"?

Reply:
Nobody want tot change it. I would say that it has to be enforced. All code files higlight _OpenTelemetry Authors_ as authors. As I remember it is done by OTel guidance.

Reply:
I think copyright notice in the source code is a different thing than the authors tag in the package metadata. Also, OpenTelemetry's guidance allow retaining previous copyright FYI https://github.com/open-telemetry/community/blob/c77b41e023291ac48ce325c6defc9783a72d4b5c/guides/contributor/processes.md?plain=1#L202-L206

Reply:
There were no such requests. We could handle it when needed : ).

IMO, if it was donated/vendore, it will be modified by someone from donators. In this case Third-party-file is good place to put all important credits.